src/main/scala/scala/tools/refactoring/implementations/ClassDefOrganizeImports.scala

```diff
@@ -0,0 +1,8 @@
+package scala.tools.refactoring
+package implementations
+
+trait ClassDefOrganizeImports { self: OrganizeImports =>
```

Comment:
as I said earlier, I'm going to move to to OrganizeImports.scala and alternatively move out Participants into separate file